packages/celo-eslint/index.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    jest: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'plugin:workspaces/recommended',
```

Comment:
Wouldn't we need `'plugin:prettier/recommended',` too? Or do we get that from the `prettier` plugin below?

Reply:
I just mentioned because thought it was meant to override the eslint formatting issues, but haven't looked much into it.